src/components/ProjectFormular.vue

```diff
@@ -137,52 +215,106 @@ export default {
+      this.validationErr.city = false
+      this.validationErr.country = false
+      this.validationErr.shorthand = false
 
       // validate fields:
       if (!this.formData.name) this.validationErr.name = true
-      if (!this.formData.email && !isValidEmail(this.formData.email))
+
+      if (!this.formData.email || !isValidEmail(this.formData.email))
         this.validationErr.email = true
-      if (!this.formData.phone && !isValidPhoneNumber(this.formData.phone))
+
+      this.formData.phone = this.formData.phone.split(' ').join('')
+      if (!this.formData.phone || !isValidPhoneNumber(this.formData.phone))
         this.validationErr.phone = true
 
+      if (!this.formData.street) this.validationErr.street = true
+      if (!this.formData.postalcode) this.validationErr.postalcode = true
+      if (!this.formData.city) this.validationErr.city = true
+      if (!this.formData.country) this.validationErr.country = true
+      if (!this.formData.shorthand) this.validationErr.shorthand = true
+
       // check for errors, no errors -> form is valid
-      if (!this.validationErr.name && !this.validationErr.email && !this.validationErr.phone)
+      if (
+        !this.validationErr.name &&
+        !this.validationErr.email &&
+        !this.validationErr.phone &&
+        !this.validationErr.street &&
+        !this.validationErr.postalcode &&
+        !this.validationErr.city &&
+        !this.validationErr.country &&
+        !this.validationErr.shorthand
+      )
         this.formValid = true
     },
     async onSubmit() {
       this.errorMessage = false
       this.successMessage = false
 
       this.validate()
+
+// reminder to check page 2 if submit is not working but page 2 has no errors
+      if(
+         this.currentPage === 2 
+      && this.validationErr.name
+      || this.validationErr.phone
+      || this.validationErr.email
+      || this.validationErr.shorthand)
+     this.pageOneErr = true
+     setTimeout(() => {
+            this.pageOneErr = false
+          }, 5000)
+
       if (this.formValid) {
         let project = {
           organisationalId: 'null',
           name: this.formData.name,
+          description: this.formData.description,
           email: this.formData.email,
-          phone: this.formData.phone
+          phone: this.formData.phone,
+          street: this.formData.street,
+          postalCode: this.formData.postalcode,
+          city: this.formData.city,
+          country: this.formData.country,
+          shorthand: this.formData.shorthand
         }
 
         try {
           await this.projectStore.setProject(project)
         } catch (error) {
           console.error(error)
+          
           // Showing error message just for 5 seconds
           this.errorMessage = true
           setTimeout(() => {
             this.errorMessage = false
           }, 5000)
+          // stop fetching to avoid loop:
+          this.projectStore.fetching = false
```

Comment:
this could not be right. If there is a timeout problem we have to handle it directly in the store or check the backend for errors.